checkstyle.xml

```diff
@@ -0,0 +1,438 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's the relationship of the file with this [one](https://github.com/Tencent/TubeMQ/blob/master/codestyle/intellij-java-google-style.xml)?

Reply:
Removed